app/database/operator/server_data_operator/handlers/post.ts

```diff
@@ -193,12 +192,6 @@ const PostHandler = (superclass: any) => class extends superclass {
             batch.push(...postFiles);
         }
 
-        if (metadatas.length) {
```

Comment:
For my understanding, why is this being removed?

Reply:
Because handleMetadata is not even defined nor used anywhere, luckily the statement always returned false

Reply:
Ok I saw that `handlePostMetadata` was removed last year.  All good :)